depmapomics/config_global.py

```diff
@@ -420,56 +424,88 @@
     ],
 }
 
+MAF_COL = "depmap_maf"
+HGNC_MAPPING = "data/new_and_old_hgnc_symbols.txt"
+
 MUTCOL_DEPMAP = [
-    "Hugo_Symbol",
-    "Entrez_Gene_Id",
-    "NCBI_Build",
-    "Chromosome",
-    "Start_position",
-    "End_position",
-    "Strand",
-    "Variant_Classification",
-    "Variant_Type",
-    "Reference_Allele",
-    "Tumor_Allele",
-    "dbSNP_RS",
-    "dbSNP_Val_Status",
-    "Genome_Change",
-    "Annotation_Transcript",
+    "chrom",
+    "pos",
+    "ref",
+    "alt",
+    "af",
+    "ref_count",
+    "alt_count",
+    "gt",
+    "ps",
+    "variant_type",
+    "variant_info",
+    "dna_change",
+    "protein_change",
+    "hugo_symbol",
+    "hgnc_name",
+    "hgnc_family",
+    "transcript",
+    "transcript_exon",
+    "transcript_strand",
+    "uniprot_id",
+    "str",
+    "dbsnp_id",
+    "dbsnp_filter",
+    "issues",
+    "gc_content",
+    "lineage_association",
+    "achilles_top_genes",
+    "cancer_molecular_genetics",
+    "ccle_deleterious",
+    "structural_relation",
+    "cosmic_hotspot",
+    "cosmic_overlapping_mutations",
+    "associated_with",
+    "lof",
+    "driver",
+    "likely_driver",
+    "transcript_likely_lof",
+    "civic_id",
+    "civic_description",
+    "civic_score",
+    "popaf",
+    "likely_gof",
+    "likely_lof",
+    "hess_driver",
+    "hess_signture",
+    "cscape_score",
+    "dann_score",
+    "revel_score",
+    "funseq2_score",
+    "pharmgkb_id",
+    "dida_id",
+    "dida_name",
+    "gwas_disease",
+    "gwas_pmid",
+    "gtex_gene",
```

Comment:
Do you want to add CamelCase renaming?

Reply:
Yes I think that would be good but I'm not sure if it's a good idea to change column names after the dry run. I will check with the portal team